stdlib/messaging/rabbitmq/src/main/java/org/ballerinalang/messaging/rabbitmq/util/ConnectionUtils.java

```diff
@@ -83,6 +108,82 @@ public static Connection createConnection(MapValue<String, Object> connectionCon
         }
     }
 
+    /**
+     * Creates and retrieves the initialized SSLContext.
+     *
+     * @param secureSocket secureSocket record.
+     * @return Initialized SSLContext.
+     */
+    private static SSLContext getSSLContext(MapValue secureSocket) {
+        try {
+            MapValue cryptoKeyStore = secureSocket.getMapValue(RabbitMQConstants.RABBITMQ_CONNECTION_KEYSTORE);
+            MapValue cryptoTrustStore = secureSocket.getMapValue(RabbitMQConstants.RABBITMQ_CONNECTION_TRUSTORE);
+
+            char[] keyPassphrase = cryptoKeyStore.getStringValue("password").toCharArray();
+            String keyFilePath = cryptoKeyStore.getStringValue("path");
```

Comment:
Shall we define constants for 'password' and 'path'?

Reply:
Resolved.